doc/man3/EVP_MD_fetch.pod

```diff
@@ -140,10 +158,14 @@ an EVP_MD object, or NULL on error.
 Fetch any available implementation of SHA256 in the default context:
 
  EVP_MD *md = EVP_MD_fetch(NULL, "SHA256", NULL);
+ ...
+ EVP_MD_meth_free(md);
```

Comment:
This is not to do with this PR. But this name meth_free name doesnt look so nice.